tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_drag_drop.py

```diff
@@ -19,7 +19,6 @@ def setUp(self):
 
     def test_drag_drop(self):
 
-        self.take_screenshot()
```

Comment:
yup, I realized there is no need to take the screenshot before drag & drop action, and it is just a cause for error because depending on how fast the screen boots up, it will show inconsistent results.